### PR TITLE
configure event queue size from env var

### PR DESCRIPTION
**Pull request by lixingwang**
**What kind of change does this PR introduce?** (check one with "x")
```
[*] Bugfix
[] Feature
[] Code style update (formatting, local variables)
[] Refactoring (no functional changes, no api changes)
[] Other... Please describe:
```

**Fixes**: #209

**What is the current behavior?**
Today we hardcode event queue size to 100 which is not good. For high loading requests, the 100 queues might be the bottleneck.

**What is the new behavior?**
We should able to configure the value from env vars.

### CODE REVIEW

File: engine/event/env.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package event
+
+import (
+	"os"
+	"strconv"
+)
+
+const (
+	EnvKeyEventQueues  = "FLOGO_EVENT_QUEUES"
```
Comment (fm-tibco) [inline engine/event/env.go:9]: Should probably be FLOGO_EVENT_QUEUE_SIZE, and corresponding methods and vars should also reflect that.

Comment (lixingwang) [reply engine/event/env.go:9]: @fm-tibco  Actually I follow the way we do for Worker and Queues. 
FLOGO_RUNNER_QUEUE
FLOGO_RUNNER_WORKERS

Do you think we should keep follow or change to FLOGO_EVENT_QUEUE_SIZE?  I agree that adding size more explicit.

Comment (fm-tibco) [reply engine/event/env.go:9]: We should deprecate FLOGO_RUNNER_QUEUE, and make it FLOGO_RUNNER_QUEUE_SIZE and use FLOGO_EVENT_QUEUE_SIZE.  I think WORKERS is ok for now.

File: engine/event/events.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ func (ec *Context) GetEventType() string {
 }
 
 // Buffered channel
-var eventQueue = make(chan *Context, 100)
+var eventQueue = make(chan *Context, GetEventQueues())
```
Comment (fm-tibco) [inline engine/event/events.go:28]: This should be GetEventQueueSize()

**Pull request merged**